core/chains/evm/config/toml/config.go

```diff
@@ -521,7 +521,7 @@ func (a *Automation) setFrom(f *Automation) {
 type Workflow struct {
 	FromAddress      *types.EIP55Address `toml:",omitempty"`
 	ForwarderAddress *types.EIP55Address `toml:",omitempty"`
-	DefaultGasLimit  uint64
+	DefaultGasLimit  *uint64
```

Comment:
Let's update this name before it goes in to a release:
```suggestion
	GasLimitDefault  *uint64
```
https://github.com/smartcontractkit/chainlink/pull/14214#discussion_r1729447663

Reply:
sorted